### DISCUSSION TITLE
The mass balance issue in whole-body PBPK due to the use of blood reservoirs?

### DISCUSSION THREAD

**Discussion by intrinsicclearance** [Open-Systems-Pharmacology/Forum#1112] [Questions & Problems]
A recent paper discussed the use of blood reservoirs in full PBPK causing the breaking of the mass balance principle.
What do you think its impact on PK-sim?

Please see Section 5 of A myth of the well-stirred model: Is the well-stirred model good for high clearance drugs?
https://doi.org/10.1016/j.ejps.2022.106134

**Comment by tobiasK2001** [comment]
Hi,

yes that seems to be an interesting question. You might want to try to find an answer going along this lines: Try to describe the Lidocaine data from Pang and Rowland, 1977 with PK-sim using their Clint input. Compare simulated values to their measurements. Make a sensitivity analysis including e.g. liver, organ blood volumes and other critical parameters like BPR, lipophilicity, organ blood flow. Check which of these parameters have an impact and how it needs to be changed to get the simulation to describe measured values.

Best, Tobias

**Reply by intrinsicclearance** [reply]
Hi Tobias,

Thank you for your response!

I guess there are some misunderstanding. I am more concerned on the discussion whether the differential equations in PBPK modeling holding the principle of mass balance in Section 5 of the paper but not the lidocaine discussions. It seems the lidocaine discussion was for a different purpose.

If I understand correctly the authors argued that the commonly used PBPK models do not hold mass balance under transient state due to the handling of the vascular space ie the introduction of the blood reservoirs when under non-steady state. And they proposed to remove the blood reservoirs so that the model follows the mass balance. In addition, the Kp equations may need to be updated to reflect the corresponding mathematical equation. 

I thought mass balance is the base of PBPK so wondering how everyone thinks about this argument.
